### PR TITLE
LPS-117466 Test if text field gets disabled when readOnly is true

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/LayoutProvider.es.js [hunk 1]
```diff
@@ -465,6 +465,49 @@ describe('LayoutProvider', () => {
...
+
+				dispatch('fieldAdded', mockEvent);
+
+				jest.runAllTimers();
+
+				const fieldNameCreated =
+					provider.state.pages[0].rows[0].columns[0].fields[0]
+						.fieldName;
+
+				const mockEvent2 = {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/LayoutProvider.es.js:495]: @alinedoleron, please re-use the variable `mockEvent`.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/LayoutProvider.es.js [hunk 2]
```diff
@@ -465,6 +465,49 @@ describe('LayoutProvider', () => {
...
+					},
+				};
+
+				const {dispatch} = child.context;
+
+				dispatch('fieldAdded', mockEvent);
+
+				jest.runAllTimers();
+
+				const fieldNameCreated =
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/LayoutProvider.es.js:491]: @alinedoleron, please rename to `expectedFieldName`.

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/natocesarrego/liferay-portal/pull/736. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/9bfeacfc9fa6~1...9bfeacfc9fa6) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**